dashboard/src/main/resources/templates/macros.ftl

```diff
@@ -52,7 +52,9 @@
   <#list jarsInProblem?values as jarInProblem>
     <@showDependencyPath dependencyPathRootCauses classPathResult jarInProblem />
   </#list>
-  <@showDependencyPath dependencyPathRootCauses classPathResult classPathEntry />
```

Comment:
In the dashboard for libraries-bom 1.0.0, guava-jdk5 was appearing twice, because it involves both source and target.